src/app/services/data/courses.data.ts

```diff
@@ -36,7 +36,7 @@ export const courses: Course[] = [
   },
   {
     id: '2',
-    title: 'JS / Front-end course EN',
+    title: 'JS / Front-end EN',
```

Comment:
I think it would be a good idea to refactor this component at all. I can see what can be done when doing this task https://github.com/rolling-scopes/site/issues/143